README.md

```diff
@@ -1,13 +1,26 @@
-# Introduction
+# `slp-parser-js`
+
+[![npm version](http://img.shields.io/npm/v/slp-parser-js.svg?style=flat)](https://npmjs.org/package/slp-parser-js "View this project on npm")
+
 This is the official .slp file parsing library. It parses a .slp file into structured data and can be used to compute stats. There are already many built-in stats that are computed by the library but the data provided can also be used to compute your own stats.
 
-# Quick-Start
-## Requirements
+## Quick-Start
+
+### Requirements
+
 * node – https://nodejs.org/en/download/
-## Writing a simple script
+
+### Installation
+
+```bash
+npm install slp-parser-js
+```
+
+### Writing a simple script
+
 1) Create a fresh directory on your disk
-1) Inside this new directory, create a file called `script.js`
-1) Fill the `script.js` file with the following contents:
+2) Inside this new directory, create a file called `script.js`
+3) Fill the `script.js` file with the following contents:
```

Comment:
This isn't necessary right? Github already auto-increments list numbers, makes it easier to inject new items in between existing list items.

Reply:
That makes the assumption that users will only see the README on Github and all other programs which can open READMEs will also auto-increment lists. People could be viewing the README locally as a text file or something that's not Github. I personally think being correct (i.e. it is a list of steps which should increment at each step) outweighs the "ease of injecting new items" but that's just me. Will revert if you want.